docker-compose.yml

```diff
@@ -9,6 +9,7 @@ web:
   environment:
     PG_USERNAME: postgres
     PG_HOST: db
+  command: bundle exec unicorn -l 0.0.0.0:3000
```

Comment:
After removing cached images and containers, I'm able to run it with just the Dockerfile CMD, so we should remove the line from compose. To remove all containers, issue ```docker rm -f `docker ps -a -q` ``` 
and to get rid of all images, ```docker rmi -f `docker images -q` ```